sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/AnalyzeTextsImpl.java

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.ai.textanalytics.implementation;
+
+import com.azure.ai.textanalytics.implementation.models.AnalyzeTextJobState;
+import com.azure.ai.textanalytics.implementation.models.AnalyzeTextJobsInput;
+import com.azure.ai.textanalytics.implementation.models.AnalyzeTextsCancelJobResponse;
+import com.azure.ai.textanalytics.implementation.models.AnalyzeTextsSubmitJobResponse;
+import com.azure.ai.textanalytics.implementation.models.ErrorResponseException;
+import com.azure.core.annotation.BodyParam;
+import com.azure.core.annotation.ExpectedResponses;
+import com.azure.core.annotation.Get;
+import com.azure.core.annotation.HeaderParam;
+import com.azure.core.annotation.Host;
+import com.azure.core.annotation.HostParam;
+import com.azure.core.annotation.PathParam;
+import com.azure.core.annotation.Post;
+import com.azure.core.annotation.QueryParam;
+import com.azure.core.annotation.ReturnType;
+import com.azure.core.annotation.ServiceInterface;
+import com.azure.core.annotation.ServiceMethod;
+import com.azure.core.annotation.UnexpectedResponseExceptionType;
+import com.azure.core.http.rest.Response;
+import com.azure.core.http.rest.RestProxy;
+import com.azure.core.util.Context;
+import reactor.core.publisher.Mono;
+
+import java.util.UUID;
+
+/** An instance of this class provides access to all the operations defined in AnalyzeTexts. */
+public final class AnalyzeTextsImpl {
```

Comment:
I figure it out! (maybe you already knew it lol) but I went and merged the operations into one class to make things easier. 
In case you want to do the same => https://github.com/Azure/azure-sdk-for-net/pull/28340

I took the idea from JS. Thanks @deyaaeldeen